### PR TITLE
Add slots for tracking status

### PR DESCRIPTION
**Pull request by deepakunni3**
This PR adds new slots pertaining to release status, submission status, and deprecation status.

### CODE REVIEW

File: src/schema/ghga.yaml [hunk 1]
```diff
@@ -1626,3 +1691,21 @@ enums:
...
+  submission_status_enum:
+    description: >-
+      Enum to capture submission status.
+    permissible_values:
+      in progress:
+        description: Signifies that a submission is in progress.
+      submitted:
+        description: Signifies that a submission is complete and submitted.
+
+  release_status_enum:
```
Comment (krausf) [inline src/schema/ghga.yaml:1704]: Maybe in the future we could add an "embargo" property, that indicates when a researcher wants to hold back data for a certain time

**Review by krausf — APPROVED** [review]
Only one question: Should GHGA-overview.png reflect the actual schema?
Other than that nothing to complain ;)

File: src/schema/ghga.yaml (hunk 1 above)
Comment (deepakunni3) [reply src/schema/ghga.yaml:1704]: I think embargo is another way of looking at `submission status` and `release status` that is communicated from the data submitter to GHGA. A separate property would be rarely used and might just be duplicative.

**Comment by deepakunni3** [comment]
Hah, yes. ghga-overview.png should reflect the actual schema. I am trying to have an autogenerator for it (not yet there yet). So I decided to remove the outdated one :)

We could also add our Miro board visualization of the model :D

**Pull request merged**